app/static/scss/admin/navbar.scss

```diff
@@ -15,12 +16,36 @@ nav {
     bottom: 0;
     background-color: $darkgray;
     color: $white;
+    @include transition(0.2s);
 
     a, a:hover {
         color: $white;
         text-decoration: none;
     }
 }
+.ham {
+    position: absolute;
+    overflow: hidden;
+    width: $nav-item-width;
+    height: $nav-item-height;
+    right: (-$nav-item-width);
+    top: 0;
+    z-index: 61;
+    display: none;
+    background-color: $darkgray;
+    .image {
```

Comment:
you can make this selector `.image, .close`, and it will apply the styles inside the`{ }` to both classes. No need to duplicate code!

(after doing this, you can delete lines 42-47.